marlowe-playground-client/src/Marlowe/Semantics.purs

```diff
@@ -1083,3 +1191,10 @@ instance hasMaxTimeArray :: HasMaxTime a => HasMaxTime (Array a) where
 
 maxOf :: Array Timeout -> Timeout
 maxOf = foldl max zero
+
+aesonCompatibleOptions :: Options
+aesonCompatibleOptions =
+  defaultOptions
+    { unwrapSingleConstructors = true
+    , sumEncoding = aesonSumEncoding
+    }
```

Comment:
why doesn't this exist in `Foreign.Generic` already, so annoying!

Reply:
Can't agree more